src/assets/projectWeekBannerAssets.tsx

```diff
@@ -2,20 +2,14 @@ import React from 'react';
 
 export const projectWeekTitle = (
   <>
-    Unsere Projektwoche: <b>Abschluss und dann?</b>
+    Unsere Fokuswoche: <b>Lernen lernen</b>
   </>
 );
 
 export const projectWeekPupilText = (
   <>
-    Vom 21.-25. März gibt es ein buntes Programm rund um das Thema{' '}
-    <b>Abschluss und dann?</b>. In verschiedenen Kursen kannst du dich mit
-    Themen rund um Studium, Ausbildung und Freiwilligendienst, aber auch
-    Bewerbungen und Studienfinanzierung auseinandersetzen.
-    <br />
-    Dies findet im Rahmen des neuen Programms &quot;Lern-Fair Fokus &ndash; die
-    digitalen Projektwochen&quot; statt, wozu du mehr Informationen auf unserer
-    Website{' '}
+    Vom 9. - 13. Mai wird ein vielseitiges Programm zum Thema “Lernen lernen” stattfinden. In verschiedenen Kursen kannst du dich über Lernstrategien, Zeitplanung, Selbstbewusstsein und vieles mehr informieren und austauschen.
+    Dies findet im Rahmen des neuen Programms Lern-Fair Fokus statt, wozu du mehr Informationen auf unserer Website
     <a
```

Comment:
in the deployment, a blank space is missing here before the link. 

From th language perspective, we could also do and remove the second half of the sentence.

Dies findet im Rahmen des neuen Programms [Lern-Fair Fokus](https://www.lern-fair.de/schueler/fokus) statt.